BE/toDoListAPI/src/main/java/school/mindera/toDoListAPI/model/DTOTaskPreview.java

```diff
@@ -9,9 +9,14 @@
 @NoArgsConstructor
 @Getter
 @Setter
-public class DTOTagPreview {
+public class DTOTaskPreview {
     private Integer taskId;
+    private Integer groupId;
     private String title;
     private Boolean isDone;
     private String date;
+    private Boolean expired;
+    private  Boolean isFavorite;
```

Comment:
```suggestion
    private Boolean isFavorite;
```
Cuidado com a formatação

Reply:
Não faltam as tags da task?